workflows/transport/pika_transport.py

```diff
@@ -233,10 +233,9 @@ def connect(self):
                 )
             try:
                 self._channel = self._conn.channel()
-            except pika.exceptions.AMQPChannelError:
-                self._conn.close()
+            except pika.exceptions.AMQPChannelError as err:
                 raise workflows.Disconnected(
-                    "Could not create channel in RabbitMQ connection"
+                    "Caught a channel error: {}, stopping".format(err)
                 )
```

Comment:
This now doesn't close the connection. And the error message seems less instructive to me. I'm not sure this change is useful? (Python will forward the inner exception anyway, so you don't have to format it)